modules/installation-configuration-parameters.adoc

```diff
@@ -705,7 +705,7 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 ifndef::openshift-origin[]
 [IMPORTANT]
 ====
-When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
+When running on ARM based AWS instances, ensure that you enter a region where AWS Graviton processors are available. Currently, AWS Graviton3 processors are only available in regions `us-east-1` and `us-west-1`. See link:https://aws.amazon.com/ec2/graviton/#Global_availability[Global availability] map in the AWS documentation.
```

Comment:
Making a detailed list this early on means that someone is going to have to be on top of keeping this up to date.  Maybe just say "only available in some regions" and rely on users following the link to see the current list?

Reply:
Yeah I think thats a great idea, I only made listed it explicitly like that bc I couldn't find a link that shows the available locations for Graviton3
@yselkowitz Do you know of a link to the AWS docs that shows the Graviton3 locations?

Reply:
the aws cli can get that info for you: `aws ec2 describe-instance-type-offerings --filters Name=instance-type,Values=c7g.xlarge`